definitive_transcendent_patch.py

```
@@ -0,0 +1,147 @@
+import sys, re
+
+def robust_sub(pattern, replacement, content, count=0, flags=0):
+    new_content, n = re.subn(pattern, replacement, content, count=count, flags=flags)
+    if n == 0:
+        print(f"CRITICAL: Pattern not found: {pattern!r}")
+        sys.exit(1)
+    return new_content
+
+def patch_main_scene():
+    path = 'src/scripts/scenes/MainScene.js'
+    with open(path) as f: content = f.read()
```

Ruff (0.15.2)

[error] Multiple statements on one line (colon)

(E701)